docs/index.md

```diff
@@ -298,3 +298,11 @@ To get started, see the [Provider Example Configs](https://github.com/couchbasec
 
   Manage Private Endpoints
 
+* [Manage Azure Network Peer](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/main/examples/network_peer)
+
+  Manage Azure Network Peer
+
+* [Manage Flush Bucket](https://github.com/couchbasecloud/terraform-provider-couchbase-capella/tree/main/examples/flush_bucket)
+
+  Manage Flush Bucket
```

Comment:
1. There are two top-level headings in this document called **Example Usage**. To differentiate, rename this section **Links to Further Examples** or similar.

2. The top few entries in this list have more description under the link. The later entries, including these two, just repeat the link title. Could expand the descriptions for later entries including these two. Little point in just repeating the link title text.

3. The top few entries in this list have a colon (:) after the link. For consistency, either remove the colon from the top few entries or add a colon to all subsequent entries, including these two.

Reply:
The description text needs to be fixed broadly. Started discussion in slack.